uaclient/cli.py

```diff
@@ -173,36 +185,32 @@ def action_disable(args, cfg):
         return 1
 
 
+@assert_attached_root
 def action_enable(args, cfg):
     """Perform the enable action on a named entitlement.
 
     @return: 0 on success, 1 otherwise
     """
+    logging.debug(ua_status.MESSAGE_REFRESH_ENABLE)
+    if not contract.request_updated_contract(cfg):
```

Comment:
Do we need to file an issue to capture the fact we were considering not doing this here?

Reply:
already filed. except that the issue morphed into, we are doing this, but we will make it more performant. https://github.com/CanonicalLtd/ubuntu-advantage-client/issues/399